docs/source/Install/installOnWindows.rst

```diff
@@ -17,10 +17,10 @@ In order to run Basilisk, the following software will be necessary:
    program from the command line
 -  `Python <https://www.python.org/downloads/windows/>`__ 3.8.x or greater
 -  `pip <https://pip.pypa.io/en/stable/installing/>`__
--  Visual Studios 15 or Greater
+-  Visual Studios 15 or greater
```

Comment:
Here you mention Visual Studio 15, but above, you say that you built using Visual Studio Community 2107 or greater. Should we standardize it, or does it not matter?

Reply:
Thank you for  the attention to detail! I've adjusted them both to fully name the version e.g. Visual Studio 15 2017 and so on.